### PR TITLE
Add hex version badge to the Readme

### PR DESCRIPTION
**Pull request by herenowcoder**
привет,

this is to have a quick link to hex. Which saves some clicks.

cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Msgpax [![Build Status](https://travis-ci.org/lexmag/msgpax.svg)](https://travis-ci.org/lexmag/msgpax)
+# Msgpax 
+[![Build Status](https://travis-ci.org/lexmag/msgpax.svg)](https://travis-ci.org/lexmag/msgpax)
+[![Hex version](https://img.shields.io/hexpm/v/msgpax.svg)](https://hex.pm/packages/msgpax)
```
Comment (lexmag) [inline README.md]: Could you capitalize `version` to be consistent with `Status`? :bowtie:

**Comment by lexmag** [comment]
I usually against having too many badges, but this one feels very useful.
Спасибо. :yellow_heart:

**Comment by herenowcoder** [comment]
конечно. I find it useful for 2 reasons:

* it provides a back link to hex
* it serves as an immediate hint for the brain that the project is hex'ed, it's faster for an eye than to vgrep readme for a line which tells how to install (if there's one).

Last but not least, thx for making Msgpax alive.

**Comment by herenowcoder** [comment]
Ehh, I think I misclicked 'close pull request' here, reopening in the case it wouldn't give you the opportunity to merge.

**Comment by lexmag** [comment]
Please squash everything into single commit.

**Comment by lexmag** [comment]
Merged to master in bae7d8424dbc74b69c1ed785d3e4defcd2497e71.

**Pull request closed without merging**